examples/svelte/package.json

```diff
@@ -13,7 +13,6 @@
     "svelte-loader": "^3.1.5",
     "svelte-preprocess": "^5.0.1",
     "tslib": "^2.5.0",
-    "typescript": "^4.9.4",
```

Comment:
I don't think it's a good choice to remove `TypeScript` from the dependency list as people may copy and paste the example out hoping for running independently.